apps/judicial-system/web/src/routes/Judge/CourtRecord/CourtRecord.tsx

```diff
@@ -89,6 +89,10 @@ export const CourtRecord: React.FC = () => {
     const defaultCourtAttendees = (wc: Case): string => {
       let attendees = ''
 
+      if (wc.judge) {
+        attendees += `${wc.judge?.name} ${wc.judge?.title}\n`
```

Comment:
```suggestion
        attendees += `${wc.judge.name} ${wc.judge.title}\n`
```

Question marks not necessary after null check above. Actually, the same applies to the prosecutor below.